src/components/UserAboutMeInputs/UserAboutMeInputs.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import staticText from "../../languages/en.pl";
+import {Textarea} from "../Formik/Textarea/Textarea";
+
+const userPageText = staticText.userPage;
+
+export const UserAboutMeInputs = () => (
+    <div className="user-page__col3">
+        <h2>{userPageText.columnTitle.aboutMe}</h2>
+        <Textarea classType={"user-page"} label={userPageText.textarea.bio.label} name={"bio"} rows={4} placeholder={userPageText.textarea.bio.placeholder}/>
+        <Textarea classType={"user-page"} label={userPageText.textarea.portfolioUrls.label} name={"portfolioUrls"} rows={4} placeholder={userPageText.textarea.portfolioUrls.placeholder}/>
```

Comment:
portolioUrls and projectUrls I think should be list of <a> elements to be clickable if HR want to visit this link

Reply:
That is not that view, here you have user page where user can adjust his info details